int32 frontRight
int32 frontLeft
int32 backRight
int32 backLeft
